EpikCord/ext/tasks.py

```diff
@@ -0,0 +1,102 @@
+import datetime
+from typing import Callable, Optional
+from time import time as _time
```

Comment:
Rename import should be avoid for clarity, i dont see where is the name conflct?

Reply:
will implement the changes just now